docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
```

Comment:
Should we have a linux make file? That way anyone can build or eventually we could set up a continuous build ...

Reply:
Linux Makefile is at /docs/Makefile.